jasmine/spec/inverted-index-test.js

```diff
@@ -0,0 +1,139 @@
+const books = require('./books.json');
+const books2 = require('./books2.json');
+const books3 = require('./books3.json');
+
+describe('InvertedIndex Class', () => {
+  beforeAll(() => {
+    invertedIndex = new InvertedIndex();
+  });
+
+  describe('Constructor', () => {
+    it('can create inverted index instance', () => {
+      expect(typeof invertedIndex).toEqual('object');
+      expect(invertedIndex instanceof InvertedIndex).toBe(true);
+    });
+
+    it('has an indexes object to hold all indexes', () => {
+      expect(typeof invertedIndex.indexes).toEqual('object');
+    });
+  });
+
+ describe('GetWords', () => {
+    it('should return an array of words', () => {
+      expect(InvertedIndex.getWords(books[0].title))
+        .toEqual(['alice', 'in', 'wonderland']);
+    });
+
+    it('filters out symbols', () => {
+      expect(InvertedIndex.getWords('alice # in* Won@derland'))
+        .toEqual(['alice', 'in', 'wonderland']);
+    });
+  });
+
+  describe('CreateIndex', () => {
+    beforeAll(() => {
+       invertedIndex.createIndex('books', books);
+       invertedIndex.createIndex('books2', books2);
+    });
+    it('creates an index', () => {
+      expect(invertedIndex.getIndex('books')).toBeTruthy();
+      expect(invertedIndex.getIndex('books2')).toBeTruthy();
+    });
+    it('creates the correct index', () => {
```

Comment:
You should have more extensive tests eg checks that return [0,2], [1,2] and if no word is found